resources/librarian/meta.yaml

```diff
@@ -0,0 +1,13 @@
+id: librarian
+handler: ansible
+version: 0.0.1
+input:
+    modules:
```

Comment:
Can we somehow make this dynamic ? Like parse manifests on runtime / resource creation ?